app/models/bank_account.rb

```diff
@@ -9,6 +9,17 @@ class BankAccount < ActiveRecord::Base
 
   scope :by_user, -> (user) { where(user_id: user.id) }
 
+  def self.setup_with_authorization_documents
+    bank_account = BankAccount.new
+
+    [:bank_authorization, :organization_authorization].each do |description|
```

Comment:
What do you think of using?
```Ruby
AuthorizationDocument.descriptions.each do |description|
  ...
end
```

It could be better when we need to add a new `description` in the model.